rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -127,29 +148,42 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
   virtual ~LifecycleNode();
 
   /// Get the name of the node.
-  // \return The name of the node.
+  /**
+   * \return The name of the node.
+   */
   RCLCPP_LIFECYCLE_PUBLIC
   const char *
   get_name() const;
 
-  /// Get the namespace of the node.
-  // \return The namespace of the node.
+  /// Get the namespace of the node
+  /**
+   * \return The namespace of the node.
+   */.
```

Comment:
```suggestion
   */
```
This is breaking your build.